_deploy/r/gnoswap/gns/gns.gno

```diff
@@ -109,6 +109,11 @@ func Mint(address pusers.AddressOrName) {
 	println("GNS_nowHeight", nowHeight)
 	println("GNS_now-lastMint", nowHeight-lastMintedHeight)
 
+	if lastMintedHeight >= nowHeight {
+		println("GNS_only one mint for each block")
```

Comment:
Does `println` function also affects gas cost? Generally, since IO functions are expensive (even if not in terms of gas). I think it might be good to comment them out later.

Reply:
![image](https://github.com/user-attachments/assets/4aba856f-a4c0-439d-9a02-4224d176b70a)

```
$ gno test -root-dir ./ -print-runtime-metrics=true  ./examples/gno.land/r/demo/echo
---       runtime: cycle=381 imports=12 allocs=35.9k(0.01%)

$ gno test -root-dir ./ -print-runtime-metrics=true  ./examples/gno.land/r/demo/echo_with_print
---       runtime: cycle=381 imports=12 allocs=35.9k(0.01%)
```

At least testing with println does not spend any more resource

Reply:
For now, we don't have anyway to get live data from chain it for debugging. so I had to print everything we need.